src/fs/directory.rs

```diff
@@ -2,6 +2,19 @@ use crate::runtime::driver::op::Op;
 use std::io;
 use std::path::Path;
 
+/// Creates a directory on the local filesystem.
+///
+/// Returns unit on success, and ['std::io::Error'] on failure.
```

Comment:
I don't think this is worthy of documentation; it's literally the signature.

Reply:
Agreed.